python/mp/rosary.py

```diff
@@ -82,7 +132,9 @@ def __init__(self, ip="127.0.0.1", port=5005, dispatcher=None, name="rosary"):
             'even_all': frozenset(self.beads[0:60:2]),
             'even_ring': frozenset(self.beads[4:60:2]),
             'odd_all': frozenset(self.beads[1:60:2]),
-            'odd_ring': frozenset(self.beads[5:60:2])
+            'odd_ring': frozenset(self.beads[5:60:2]),
+            'base': frozenset(self.bases[0:self.BASE_COUNT]),
+            'cross': frozenset(self.cross[0:self.CROSS_LED_COUNT])
         }
```

Comment:
two new set names: base and cross. we're going to want more, but this gets us started.